netbox/virtualization/api/serializers.py

```diff
@@ -84,17 +84,24 @@ class VirtualMachineSerializer(NetBoxModelSerializer):
 
     # Counter fields
     interface_count = serializers.IntegerField(read_only=True)
+    virtual_disk_count = serializers.IntegerField(read_only=True)
 
     class Meta:
         model = VirtualMachine
         fields = [
             'id', 'url', 'display', 'name', 'status', 'site', 'cluster', 'device', 'role', 'tenant', 'platform',
             'primary_ip', 'primary_ip4', 'primary_ip6', 'vcpus', 'memory', 'disk', 'description', 'comments',
             'config_template', 'local_context_data', 'tags', 'custom_fields', 'created', 'last_updated',
-            'interface_count',
+            'interface_count', 'virtual_disk_count', 'disk_space',
```

Comment:
We should figure out a way to annotate `disk_space` on each instance, to avoid calling a separate query per VM on list view.